mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -3635,6 +3636,63 @@ static uint64_t getTeamsReductionDataSize(mlir::omp::TeamsOp &teamsOp) {
   return getReductionDataSize<mlir::omp::TeamsOp>(teamsOp);
 }
 
+static void
+extractHostEvalClauses(OperandRange ops, llvm::ArrayRef<BlockArgument> args,
+                       Value &numThreads, Value &numTeamsLower,
+                       Value &numTeamsUpper, Value &threadLimit,
+                       llvm::SmallVectorImpl<Value> *lowerBounds = nullptr,
+                       llvm::SmallVectorImpl<Value> *upperBounds = nullptr,
+                       llvm::SmallVectorImpl<Value> *steps = nullptr) {
+  for (auto item : llvm::zip_equal(ops, args)) {
+    Value op = std::get<0>(item), arg = std::get<1>(item);
```

Comment:
Nit: I generally refrain from making comments about variable names but in this case could you consider renaming `op` and `arg` to `hostEvalVar` and `blockArg` or something on these lines?